.changeset/gold-yaks-grab.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-tokens": minor
+---
+
+Add core.shadow token. Export modal tokens.
```

Comment:
Do we need to update the changeset since we don't add a new shadow token anymore?

Reply:
This was to make sure that I would include the CSS vars generated for WB Modal for all the 3 stacked PRs. But turns out that I didn't land this last PR with the other two, so I messed up the release (and probably this is not needed anymore).

Reply:
If we keep the changeset, could we remove the `Add core.shadow token` part since we no longer add the `instructive` shadow token?